yocto/runqemu-smoke-test.sh

```diff
@@ -42,8 +42,5 @@ bitbake qemu-helper-native
 # Set up networking for qemu
 sudo ../sources/poky/scripts/runqemu-gen-tapdevs `id -u` `id -g` 4 tmp/sysroots-components/x86_64/qemu-helper-native/usr/bin
 
-# Give qemu permissions to access tun interfaces
-sudo setcap cap_net_admin+ep $(find tmp/work/ -name qemu-system-x86_64 | grep qemu-helper-native)
```

Comment:
For some reason which is not explained very well in https://github.com/Pelagicore/vagrant-cookbook/commit/92ca2467b71539b619fa4d76bc1cdb84096534da, there was a need to configure the permission for the tun interface.
And now, when this quirk is removed, I wonder if there are any tests that will be broken? Perhaps the SoftwareContainer tests?

However, what I can see in the build log, https://pelux.io/jenkins/job/pelux-manifests_SMOKE_TEST/440/, is that the software container tests fails, and this is due to the fact that SC has been temporarily disabled in this commit: https://github.com/Pelagicore/meta-pelux/commit/94ad5647c9834cc4156780f77f53b5fc5c3ce3f4

Therefore, I would like that SC can be enabled again and see if this PR affects the smoke test in a negative way or not.

Reply:
The reason is most likely that someone didn't understand that '1000' is in fact the user/group id that owns the tun interfaces, and it must match with the user running the script for qemu to work. So instead of fixing the script to supply the actual uid/gid that matches tun interfaces, they added a hack to give qemu the needed capability via setcap. As I explained, this breaks correct loading of libraries on the host system; we are affected badly by this in the SW Force Mbient project. That qemu is able to start in your environment is a coincidence.

None of this has anything to do with the tests that run inside qemu on the guest os; they should not be affected at all.

Reply:
@jonte ^^^ please note.

Reply:
> The reason is most likely that someone didn't understand that '1000' is in fact the user/group id that owns the tun interfaces, and it must match with the user running the script for qemu to work. So instead of fixing the script to supply the actual uid/gid that matches tun interfaces, they added a hack to give qemu the needed capability via setcap. As I explained, this breaks correct loading of libraries on the host system; we are affected badly by this in the SW Force Mbient project. That qemu is able to start in your environment is a coincidence.
> 
> None of this has anything to do with the tests that run inside qemu on the guest os; they should not be affected at all.

Thanks @kanavin for the explanation. It was not clear when reading the commits, so perhaps you could further improve the commit messages to explain the reasoning behind them.

Reply:
Done - does it look ok now?